src/audio/component.c

```diff
@@ -322,10 +322,12 @@ int comp_verify_params(struct comp_dev *dev, uint32_t flag,
 
 struct comp_dev *comp_make_shared(struct comp_dev *dev)
 {
```

Comment:
@tlauda is there any bug associated with this? Would be interesting to see when this does actually is a problem.

Reply:
@dbaluta No bug. Tests will be added to CI as soon as this patch is merged.